notebooks/105-language-quantize-bert/105-language-quantize-bert.ipynb

```diff
@@ -522,10 +630,60 @@
     "tags": []
    },
    "source": [
-    "## Compare Performance of the Original and Quantized Models\n",
-    "Finally, we will measure the inference performance of the FP32 and INT8 models. To do this, we use [Benchmark Tool](https://docs.openvinotoolkit.org/latest/openvino_inference_engine_tools_benchmark_tool_README.html) - OpenVINO's inference performance measurement tool.\n",
+    "## Compare Performance of the Original, Converted and Quantized Models\n",
+    "\n",
+    "To see the performance difference we will compare original PyTorch and OpenVINO converted model (FP32)."
```

Comment:
```suggestion
    "To see the performance difference we will compare the original PyTorch model with the OpenVINO quantized model (INT8)."
```
The model loaded is `compressed_model_xml`. It makes sense to me to compare with the INT8 model too. You could also add the IR FP32 or FP16 comparison here, though that's also in benchmark_app right below this function.